setup.py

```diff
@@ -109,16 +109,7 @@ def get_long_description():
           author_email='avocado-devel@redhat.com',
           url='http://avocado-framework.github.io/',
           use_2to3=True,
-          packages=['avocado',
-                    'avocado.core',
-                    'avocado.utils',
-                    'avocado.utils.external',
-                    'avocado.core.remote',
-                    'avocado.core.restclient',
-                    'avocado.core.restclient.cli',
-                    'avocado.core.restclient.cli.args',
-                    'avocado.core.restclient.cli.actions',
-                    'avocado.plugins'],
+          packages=find_packages(exclude=('selftests*',)),
```

Comment:
I tried the `include` approach and it seems to work well on my F23: https://github.com/avocado-framework/avocado/pull/1356#discussion_r73998340

can you please reconsider the `include` rather than `exclude`? It could be hard to spot an extra package created by avocado...